generic3g/specs/StateItemSpec.F90

```diff
@@ -49,6 +53,10 @@ module mapl3g_StateItemSpec
       procedure(I_can_connect), deferred :: can_connect_to
       procedure(I_make_adapters), deferred :: make_adapters
 
+      procedure :: get_aspect_order ! as string vector
+!#      procedure(I_get_aspect_priorities), deferred :: get_aspect_priorities ! as colon-separated string
```

Comment:
Could this be deleted?

Reply:
The line after it is temporary for the bootstrapping process.   Need a default implementation for subclasses to compile.    Once bootstrapping is complete, that line will be uncommented and the next line will be deleted.